examples/vnet/doca_gw_dpdk.c

```diff
@@ -488,11 +530,29 @@ doca_gw_dpdk_build_vxlan_header(uint8_t **header,
 	*header += sizeof(vxlan_hdr);
 }
 
+static void
+doca_gw_dpdk_build_gre_header(uint8_t **header,
+		struct doca_gw_pipeline_cfg *cfg)
+{
+	uint32_t *key_data;
+	struct rte_gre_hdr gre_hdr;
+
+	memset(&gre_hdr, 0, sizeof(struct rte_gre_hdr));
+	gre_hdr.k = 1;
+	gre_hdr.proto = rte_cpu_to_be_16(RTE_ETHER_TYPE_IPV4);
+	memcpy(*header, &gre_hdr, sizeof(gre_hdr));
+	*header += sizeof(gre_hdr);
+	key_data = (uint32_t *)(*header);
+	*key_data = rte_cpu_to_be_32(cfg->match->tun.gre.key);
```

Comment:
no rte_cpu_to_be_32, user should put correctly

Reply:
done